src/main.c

```diff
@@ -441,6 +441,10 @@ static void get_input(char* in) {
 
     // Collect input until enter is pressed
     for (int pos = 0, len = 0; (inp = getchar()) != 13;) {
+
+        // Get max possible input length
+        int max = getmaxx(inputwin) - 24;
```

Comment:
Could you declare a constant for `24`? It would be better not to use magic numbers :)